packages/hub/src/components/EntityLayout.tsx

```diff
@@ -0,0 +1,40 @@
+import { FC, ReactNode } from "react";
+import { EntityInfo } from "./EntityInfo";
+import { clsx } from "clsx";
+
+type Props = {
+  slug: string;
+  username: string;
+  homepageUrl: string;
+  headerChildren: ReactNode;
+  children: ReactNode;
+  isFluid?: boolean;
+};
+
+export const EntityLayout: FC<Props> = ({
+  slug,
+  username,
+  homepageUrl,
+  headerChildren,
+  children,
+  isFluid = false,
+}) => {
+  return (
+    <div>
+      <div className="border-gray-300" style={{ backgroundColor: "#eceef0" }}>
+        <div
+          className={clsx(
+            "flex items-center gap-4 pt-5 pb-4",
+            !isFluid ? "max-w-4xl mx-auto" : "px-8"
+          )}
+        >
+          <EntityInfo slug={slug} username={username} href={homepageUrl} />
+          {headerChildren}
+        </div>
+      </div>
+      <div className={clsx(!isFluid && "max-w-4xl mx-auto my-4")}>
```

Comment:
The description column is currently too narrow with `w-4xl`. Though maybe it could be fixed with a different `grid-template-columns` value.

<img width="1070" alt="Captura de pantalla 2023-06-17 a la(s) 16 40 31" src="https://github.com/quantified-uncertainty/squiggle/assets/89368/9ce3c864-35c0-40e6-bc21-2250a5914efe">

Reply:
Hm... good point about this table. This table could really use full width.